config/config.example.toml

```diff
@@ -323,8 +323,8 @@ paypal = { currency = "USD,INR", country = "US" }
 # ^------------------------------- any valid payment method type (can be multiple) (for cards this should be card_network)
 # If either currency or country isn't provided then, all possible values are accepted
 
-# KMS configuration. Only applicable when the `kms` feature flag is enabled.
-[kms]
+# AWS KMS configuration. Only applicable when the `aws_kms` feature flag is enabled.
+[aws_kms]
```

Comment:
Can we not do this config section rename as part of this PR (considering that it will be renamed yet again when we add a runtime toggle for KMS), and restrict this PR to only renaming the feature?